src/realm/sync/client.cpp

```diff
@@ -1192,6 +1192,10 @@ bool SessionWrapper::has_flx_subscription_store() const
 void SessionWrapper::on_flx_sync_error(int64_t version, std::string_view err_msg)
 {
     REALM_ASSERT(!m_finalized);
+    // Don't report errors on server initiated bootstraps to the subscription store
+    if (m_flx_active_version == version) {
```

Comment:
this is only used to report a query error. Why would a role change bootstrap have anything to do with it?

Reply:
I don't think there's a case (yet) where a QUERY_ERROR could be received for a server initiated bootstrap - I removed this check.